docs/validators/nodes-scripts/use-docker.md

```diff
@@ -3,26 +3,53 @@ id: use-docker
 title: How to use the Docker Image
 ---
 
-As an alternative to the recommended installation flow, one could choose to run an MultiversX (previously Elrond) Node using the official Docker image: [elrondnetwork/elrond-go-node](https://hub.docker.com/r/elrondnetwork/elrond-go-node-testnet)
+As an alternative to the recommended installation flow, one could choose to run an MultiversX Node using the official Docker images: [here](https://hub.docker.com/orgs/multiversx/repositories)
 
-As an alternative to the recommended installation flow, one could choose to run an MultiversX Node using the official Docker image: [elrondnetwork/elrond-go-node](https://hub.docker.com/r/elrondnetwork/elrond-go-node-testnet)
+On the `dockerhub` there are Docker images for every chain (mainnet, devnet and testnet).
 
-Pull the latest image of an MultiversX node with this config from Dockerhub
+Images name:
+- for mainnet: [chain-mainnet](https://hub.docker.com/repository/docker/multiversx/chain-mainnet)
+- for devnet: [chain-devnet](https://hub.docker.com/repository/docker/multiversx/chain-devnet)
+- for testnet: [chain-testnet](https://hub.docker.com/repository/docker/multiversx/chain-testnet)
```

Comment:
these links requires authentication for me, like this might be better? https://hub.docker.com/r/multiversx/chain-mainnet

Reply:
fixed